README.md

```diff
@@ -10,6 +10,37 @@ intended for packagers, release engineers, testers and devlopers.
 The intention is to store all pre-compiled (and signed if neccesary) firmware
 binaries and pre-compiled topologies for snapshot, stable and LTS releases.
 
+# Installation
+
+The latest SOF release is version 1.6.1 and is available here on the
+stable-v1.6.1 branch. If you want to install this manually instead of from your
+distribution then please follow these instructions
+
+```
+git checkout stable-v1.6.1
+sudo ./go.sh
```

Comment:
@xiulipan do we actually have stable-v1.6.1 branch?

Reply:
@dbaluta No stable-v1.6.1 is indeed a branch. https://github.com/thesofproject/sof-bin/tree/stable-v1.6.1

For v1.6.1 and v1.6 even though we only has few patches, but I think the patch would be almost a re-write as most FW binary is changed.

Reply:
@xiulipan, i see so v1.6.1 is a branch in sof-bin repo but a tag in sof repo.